### PR TITLE
Fix some minor errors in the chathistory specification.

### PR DESCRIPTION
**Pull request by SaberUK**

### CODE REVIEW

**Comment by dequis** [comment]
Yeah I was wondering about the capitalization too but thought it wasn't worth making another pull request. But that's done now so :+1:

File: extensions/batch/chathistory-3.3.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ local client is in for public messages.
 When a client which has enabled the [`batch` capability][batch] performs an
 action which requires chat history to be relayed to it (e.g. joining a channel
 with InspIRCd's `chanhistory` mode enabled) the server SHOULD put all PRIVMSGs
-and NOTICEs into a single CHATHISTORY batch. If the client has also enabled the
-[`server-time` capability][server-time] capability then messages inside the
-batch MUST be tagged with the time at which they were originally sent.
+and NOTICEs into a single `chathistory` batch. If the client has also enabled
+the [`server-time` capability][server-time] then messages inside the batch MUST
```
Comment (grawity) [inline extensions/batch/chathistory-3.3.md]: Where did "capability" go?

Comment (dequis) [inline extensions/batch/chathistory-3.3.md]: Inside the link text

```diff
-[`server-time` capability][server-time] capability
+[`server-time` capability][server-time]
```

Comment (grawity) [inline extensions/batch/chathistory-3.3.md]: Ah, d'oh.

**Pull request merged**